app/src/main/java/com/example/tasky/login/domain/LoginViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.tasky.login.domain
+
+import androidx.compose.ui.text.input.TextFieldValue
+
+class LoginViewModel {
```

Comment:
You should inherit from ViewModel()

Reply:
I wanted to leave the creation and the connection with the viewModel later